src/hooks/useSnapshotDelegateContract.ts

```diff
@@ -15,12 +15,20 @@ const GLOBAL_SPACE_ID = '0x00000000000000000000000000000000000000000000000000000
 
 const validateContract = <T>(contract: Contract | undefined, callback: (contract: Contract) => Promise<T>) => {
   if (!contract) {
-    throw new Error("Delegation Contract is undefined")
+    throw { code: -1, message: "Delegation Contract is undefined" }
```

Comment:
I'd love it if we could send this -1 to a constant that adds meaning by it's name, like 
`ERROR_CODE`.

Also for `result.status !== 1` ==> `result.status !== SUCCESS_CODE`